src/tests/ui/anonymous-experiences/request.spec.ts

```diff
@@ -75,19 +75,32 @@ test.describe('Run query', () => {
     await page.locator('label:has-text("messages")').click();
     await page.evaluate(() => document.fonts.ready);
     expect(await page.screenshot({ clip: { x: 300, y: 0, width: 1920, height: 1080 } })).toMatchSnapshot();
-    await queryInputField.type('?sel');
+    await queryInputField.type('?$sel');
```

Comment:
beta doesn't require the $ symbol on odata query parameters. 
Do we have tests to validate this?

Reply:
No we do not. Let me add more tests to cover this case